CI chain ails when building docker images

Noticed on one of my PRs (*) that the docker build is failing with the following error:

```
ERROR: unsatisfiable constraints:
  git-2.24.4-r0:
    breaks: world[git=2.24.3-r0]
The command '/bin/sh -c apk --update add --no-cache git=2.24.3-r0' returned a non-zero code: 1
Failed to build Docker Image keptn/shipyard-controller:0.8.1-dev-PR-3480.202103100837, exiting
Error: Failed to build Docker Image
Error: Process completed with exit code 1.
```

(*) https://github.com/keptn/keptn/pull/3480/checks?check_run_id=2075031060

--- a/configuration-service/Dockerfile
+++ b/configuration-service/Dockerfile
@@ -42,7 +42,7 @@ ENV env=production
 RUN apk add --no-cache ca-certificates libc6-compat
 
 # Install git
-RUN apk --update add --no-cache git=2.24.3-r0
+RUN apk --update add --no-cache git=2.24.4-r0
 
 # Copy the binary to the production image from the builder stage.
 COPY --from=builder /go/src/github.com/keptn/keptn/configuration-service/main /configuration-service

